apiary.apib

```diff
@@ -371,12 +371,19 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added `budget` to `projects.list` and `projects.info`.
```

Comment:
You don't need to document it twice... adding it to the latest section is fine.
Every month we will group "latest" stuff to the specific month, so those entries stay stable and people don't have to reread them all the time..